packages/zod/src/__tests__/contract.test.ts

```diff
@@ -0,0 +1,21 @@
+import { string } from 'zod';
+
+import { zodContract } from '../zod_contract';
+
+describe('runtypes/runtypeContract short', () => {
```

Comment:
Could you add some more tests:
- for `.isData` method, now only `.getErrorMessages` is covered
- for more complex cases, like arrays or records